src/devices/cpu/xtensa/xtensa.cpp

```diff
@@ -29,43 +46,2254 @@ std::unique_ptr<util::disasm_interface> xtensa_device::create_disassembler()
 
 device_memory_interface::space_config_vector xtensa_device::memory_space_config() const
 {
-	return space_config_vector {
-		std::make_pair(AS_PROGRAM, &m_space_config),
+	space_config_vector spaces = {
+		std::make_pair(AS_PROGRAM, &m_space_config)
 	};
+
+	spaces.push_back(std::make_pair(AS_EXTREGS, &m_extregs_config));
+	return spaces;
+}
```

Comment:
Why not just put both the spaces in the return statement?  Like:
```c++
	return space_config_vector{
		std::make_pair(AS_PROGRAM, &m_space_config),
		std::make_pair(AS_EXTREGS, &m_extregs_config)
	};
```

Reply:
done